workspaces/explore/plugins/explore-backend/src/tools/types.ts

```diff
@@ -1,33 +0,0 @@
-/*
- * Copyright 2022 The Backstage Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import {
-  GetExploreToolsRequest,
-  GetExploreToolsResponse,
-} from '@backstage-community/plugin-explore-common';
-
-/**
- * @public
- */
-export interface ExploreToolProvider {
```

Comment:
Should we be perhaps deprecating this instead of removing it? Would this cause a braking change?

Reply:
I have changed it so it is reexported, so users which have custom ToolProviders have time to migrate